src/tag.mli

```diff
@@ -1,11 +1,13 @@
-val v : ?arch:Ocaml_version.arch -> ?switch:Ocaml_version.t -> Dockerfile_distro.t -> string
-(** [v ?arch ?switch distro] is the Docker tag to use for an image built on [distro] and [arch]
+val v : ?latest_distro:bool -> ?arch:Ocaml_version.arch -> ?switch:Ocaml_version.t -> Dockerfile_distro.t -> string
+(** [v ?latest_distro ?arch ?switch distro] is the Docker tag to use for an image built on [distro] and [arch]
     with OCaml compiler [switch] installed. If [switch] is [None] then this is a base image
     with no switches. If [arch] is set then this is a staging image, which will later be combined
-    into a cross-platform image. *)
+    into a cross-platform image. If [latest_distro] is true (default: false) then the distro
+    version will not be included in the tag (e.g. [alpine-ocaml-4.10]).  *)
```

Comment:
Why do we need the `latest_distro` argument? We're already passing the alias as `distro`. I think the only place this will make a difference is with e.g. `ubuntu-lts`. But in that case we really do want the alias to be `ubuntu-lts` and not `ubuntu`.

Reply:
We don't -- I forgot that the `Latest` distro already resolves to a tag without the version.